http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java

```diff
@@ -174,13 +173,9 @@ public void nonAppRedirections() {
     }
 
     @Test
-    @Disabled("blocked by: https://issues.redhat.com/browse/QUARKUS-781")
     public void microprofileHttpClientRedirection() throws Exception {
-        HealthClientService healthHttpClient = new RestClientServiceBuilder<HealthClientService>(getAppEndpoint())
-                .withHostVerified(true).withPassword(PASSWORD).withKeyStorePath(KEY_STORE_PATH)
-                .build(HealthClientService.class);
-
-        assertThat(SC_OK, equalTo(healthHttpClient.health().getStatus()));
+        io.restassured.response.Response health = getApp().given().get("api/client");
```

Comment:
could you add this import `io.restassured.response.Response` to the class and remove the full qualified name from the Response variable?

Reply:
I can't, there is already import of Response on line 49[1].

[1] https://github.com/quarkus-qe/quarkus-test-suite/blob/9963984c74f67f8d05c2603abf8204a894c7558e/http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java#L49